spec/pathwatcher-spec.coffee

```diff
@@ -114,6 +114,21 @@ describe 'PathWatcher', ->
           done()
       fs.writeFileSync(tempFile, 'changed')
 
+  describe 'when watching a file that does not exist', ->
+    it 'throws an error with a code', (done) ->
```

Comment:
(Or at least, I tried to, but after my push https://github.com/meteor/node-pathwatcher/tree/expose-errno-pr is 500'ing.  Eek.)